Bookish/Views/Catalog/BorrowerList.cshtml

```diff
@@ -0,0 +1,53 @@
+﻿@using Microsoft.EntityFrameworkCore.Metadata.Internal
+@model Bookish.Models.BorrowerViewModel
+
+@{
+    ViewBag.Title = "Borrower List";
+}
+
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css">
+    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.12.1/css/jquery.dataTables.css">
+    <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.12.1/js/jquery.dataTables.js"></script>
+
+    <title>Borrower List</title>
+</head>
+<body>
+<div>
+    <table class="display" style="width:100%" id="myTable">
```

Comment:
Try not to use inline styles but find a Bootstrap class which does what you want

Reply:
Additionally the id should probably be a bit more descriptive - what's it a table of?